File: solutions/2715-k-items-with-the-maximum-sum/solution.py
class Solution(object):
    def kItemsWithMaximumSum(self, numOnes, numZeros, numNegOnes, k):
        """
        :type numOnes: int
        :type numZeros: int
        :type numNegOnes: int
        :type k: int
        :rtype: int
        """
        ls=[1]*numOnes+[0]*numZeros+[-1]*numNegOnes
        return sum(ls[:k])

File: solutions/2715-k-items-with-the-maximum-sum/solution.java
class Solution {
    public int kItemsWithMaximumSum(int numOnes, int numZeros, int numNegOnes, int k) {
        int n=numOnes+numZeros+numNegOnes;
        int i,sum=0;
        int m=numOnes+numZeros;
        int[] arr=new int[n];
        for(i=0;i<numOnes;i++)
            arr[i]=1;
        for(i=numOnes;i<m;i++)
            arr[i]=0;
        for(i=m;i<n;i++)
            arr[i]=-1;
        for(i=0;i<k;i++)
            sum+=arr[i];
        return sum;
    }
}
